src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -846,7 +846,7 @@ private void rawDeleteHelper(RawDeleteResponse resp, TiRegion region) {
       throw new TiClientInternalException("RawDeleteResponse failed without a cause");
     }
     String error = resp.getError();
-    if (error != null && !error.isEmpty()) {
+    if (!error.isEmpty()) {
```

Comment:
it's not related to the pr title.

Reply:
They are related to similar changes in `rawDeleteRangeHelper`